hazi.html

```diff
@@ -2,78 +2,229 @@
 <html lang="hu">
     <head>
         <title>
-            Húros hangszerek
+            Vonós hangszerek
         </title>
+        <link rel="stylesheet" href="hazi.css">
     </head>   
-    <body>
-        <a id="ugras_fel"> </a> 
-        <a href="#ugras_le"> Ugrás az oldal aljára </a>   
-        <hr>
-        <h1>Hegedű</h1>
-        <hr>
-            <p>A hegedű a vonós hangszerek hegedűcsaládjának legmagasabb hangolású, méretre legkisebb tagja, 4 db, kvint távolságra hangolt húrral. A csoportba tartozik még a mélyhegedű, vagy közismertebb nevén brácsa, a cselló (más néven gordonka) és a nagybőgő (másként gordon).</p>
-            <p>A legmélyebb húr (ami tulajdonképpen a hegedűn megszólaltatható legmélyebb hang) a kis G, ezt követi az egyvonalas D-, egyvonalas A-, illetve a kétvonalas E-húr. A hegedűkották általában violinkulcsban (más néven G-kulcs) íródnak.</p>
-            <p>A hangszer iránt támasztott megnövekedett igények következtében az egyik legösszetettebb építési szakértelmet igénylő hangszer lett. Mérete, gondos készítési és játéktechnikai bánásmód kombinációja a többi húros hangszert meghaladó virtuozitást, valamint széles skálán változtatható dinamikai előadást is megengedő teljesítményt eredményez. A vonósok közül talán a legnépszerűbb, de mindenképpen a leggyakrabban előforduló és legkeresettebb hangszer.</p>
-            <h2>Etimológia</h2>
+    <body class="basic hatter">
+        <div>
+            <a id="ugras_fel"> </a> 
+            <nav class="footer_nav">
+                <a href="#ugras_le" class="link_basic"> Ugrás az oldal aljára </a>   
+            </nav>
+        </div>
+        <div class="div_border">
```

Comment:
A `div` tag elég általánosan használt, classnévbe téve pedig nem segít az átláthatóságban. Ha ráteszed egy `nav` tag-re, akkor átnevezed? Más elemre nem tehetem ezt a classt, csak divre?